playbook.md

```diff
@@ -189,24 +189,38 @@ There are several states that a story has to go through in order to be deployed
 * **Deployed**: The story is deployed to production
 
 #### Branching
-When we start any new piece of work - whether story, bug fix, or chore - we always start by creating a new branch based off the repository's `develop` branch.
+When we start any new piece of work - whether story, bug fix, or chore - we always start by creating a story in our tracker and creating a new branch based off the repository's `develop` branch.
```

Comment:
@magdamaciaszczyk Yes we should. Let's talk about Tickets at the next dev chat (this Friday).